elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpers.java

```diff
@@ -64,14 +89,19 @@ public static String formatFilePath(String basePath) {
      * @param basePath : root path to model dir
      * @return Map of variables
      * @throws JsonProcessingException
+     * @throws ProcessingException
      */
     @SuppressWarnings("unchecked")
-    public static Map<String, Object> getVariablesPojo(String basePath) throws JsonProcessingException {
+    public static Map<String, Object> getVariablesPojo(String basePath)
+            throws JsonProcessingException, ProcessingException {
         String filePath = basePath + VARIABLE_CONFIG_PATH;
         File variableFile = new File(filePath);
         if (variableFile.exists()) {
             String jsonConfig = hjsonToJson(readConfigFile(variableFile));
-            return getModelPojo(jsonConfig, Map.class);
+            if (verifySchema(VARIABLE, jsonConfig)) {
+                return getModelPojo(jsonConfig, Map.class);
+            }
+            return null;
```

Comment:
instead of multiple return we can store results in a variable,  and then have single return.